pkg/reconciler/isbsvc/installer/assets/redis/config/replica.conf

```diff
@@ -1,5 +1,4 @@
 dir /data
-slave-read-only yes
```

Comment:
Remove this because it's the default behavior?

Reply:
Yes, that and the fact that the default Helm chart doesn't set it.

Reply:
And it actually looks like it was explicitly taken out - the older chart (deploying v6.2) does include it and the newer chart (deploying v7) doesn't.